lib/config.js

```diff
@@ -30,7 +30,7 @@ var Configurator = function (file) {
   });
 };
 
-util.inherits(Configurator, process.EventEmitter);
+util.inherits(Configurator, require('events'));
```

Comment:
this means that npm install events is needed?  Also should it be "require('events').EventEmitter"?

Reply:
I guess it should; and that's how it's done in #637.